bounds:
  lower_bound: 0.1
  upper_bound: 0.2
joints:
  rotational_joint:
    minimum_torque: 0.0
    maximum_torque: 0.1
  linear_joint:
    minimum_torque: 0.0
    maximum_torque: 0.1
  
